api_tests/src/actors/swap_factory.ts

```diff
@@ -234,6 +234,12 @@ async function makePeer(actor: Actor): Promise<Peer> {
     };
 }
 
+async function makePeerIdOnly(actor: Actor): Promise<Peer> {
+    return {
+        peer_id: await actor.cnd.getPeerId(),
```

Comment:
I think the deserializer also supports not using an object here but just the peer id as value.

Reply:
I understand what you write, but have trouble achieving this in code because of the `await`. 
I could make a variable that awaits the peer-id and then return that variable, but I don't really see much value in that. 
What exactly did you have in mind?